src/transaction/boot_sr.c

```diff
@@ -2528,6 +2528,7 @@ boot_restart_server (THREAD_ENTRY * thread_p, bool print_restart, const char *db
   oid_set_root (&boot_Db_parm->rootclass_oid);
 
   /* Load and recover data pages before log recovery */
+  // TODO: only when !is_tran_server_with_remote_storage ?
   error_code = dwb_load_and_recover_pages (thread_p, log_path, log_prefix);
```

Comment:
It is already earlied out inside `dwb_load_and_recover_pages()`. It the check is moved here, it should be removed from `dwb_load_and_recover_pages()`

Reply:
I moved the check upper and removed checks from `double_write_buffer.c`. I think it is more appropriate to concentrate these checks in fewer places/functions.